abstract/abstract.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Comparing strategies for assessing uncertainty with geophysical inversions for mineral exploration'
+
+abstract: |
+    Drilling plays a crucial role in exploration programs, and geophysical data often aid in choosing drill locations. Electromagnetic methods, sensitive to subsurface conductivity, are commonly used to create subsurface models for this purpose. However, geophysical inversions are inherently non-unique, as multiple Earth models can fit the data. Therefore, the uncertainty in the obtained models is of interest. However, formal uncertainty quantification poses a challenge, given the difficulty in translating the ultimate decision into a mathematical framework. Our work is to aim at using a blend of deterministic and Bayesian methods to assess uncertainty.
```

Comment:
@rowanc1 or @stevejpurves: Whenever you have a chance, I would appreciate your help here. The abstract shows up in my local myst preview, but it doesn't seem to show up in the curvenote preview: https://www.appliedgeophysics.org/previews/0192df79-8e71-78b5-bbca-008d3ba83f34?preview=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJodHRwczovL2pvdXJuYWxzLmN1cnZlbm90ZS5jb20iLCJleHAiOjE3MzA3NTg0MjYsImF1ZCI6ImFwcGxpZWRnZW9waHlzaWNzIiwic2NvcGUiOiJzdWJtaXNzaW9uIiwic2NvcGVJZCI6IjAxOTA5OTlmLTY2NTctNzBjYi1hNTg1LTVjOTkwODY0YzIyMyIsImlhdCI6MTczMDMyNjQyNn0.gN-gGXTARTwFK2QbBlyqDuj3Bt4ZeeXTILI4-y3YNc0

Any pointers are appreciated! Thank you